### DISCUSSION TITLE
Access dockertasmocompiler behind apache proxy

### DISCUSSION THREAD

**Discussion by jeroenst** [benzino77/tasmocompiler#290] [Q&A]
I tried to proxy myserverdomain.org/tasmotizer to the running docker image on port 4000.

When I access port 4000 directly or works perfectly but when using the apache proxy I get a blank screen.

ProxyPass           /tasmocompiler   http://172.30.30.1:4000
ProxyPassReverse    /tasmocompiler   http://172.30.30.1:4000

Why isn't this working and how can I solve this?

**Comment by benzino77** [comment]
Hi Jeroen,

You should ask those kind of questions on Apache forum rather ;)

**Reply by jeroenst** [reply]
Hi!
The problem is that the path is hardcoded to /cgi-bin. I created a virtual host tasmocompiler.domainname.org, that works fine.

**Reply by benzino77** [reply]
There is no such patch (/cgi-bin) on tasmocompiler side. 
What I can say is that when you modify path for web service you have to rewrite that url on Reverse Proxy side before the traffic is directed to such service.
